### PR TITLE
[Merged by Bors] - 0.10 Blog Post: animation transition

### PR DESCRIPTION
**Pull request by mockersf**

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
Nice and simple, helpful example.

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -1170,6 +1170,62 @@ commands.spawn(SpriteBundle::default()).add(MyCustomCommand);
...
+<div class="release-feature-authors">authors: @smessmer</div>
+
+You can now smoothly transition between two (or more) skeletal animations!
+
+<video controls loop><source  src="animation_transition.mp4" type="video/mp4"/></video>
+
+<div style="font-size: 1.0rem" class="release-feature-authors">Character model and animations are royalty free assets from Mixamo.
+</div>
+
+With the new [`play_with_transition`] method on the [`AnimationPlayer`] component, you can now specify a transition duration during which the new animation will be linearly blended with the currently playing animation, which weight will decrease during that duration until it reaches `0.0`.
```
Comment (alice-i-cecile) [inline content/news/2023-03-04-bevy-0.10/index.md:1184]:
```suggestion
With the new [`play_with_transition`] method on the [`AnimationPlayer`] component, you can now specify a transition duration during which the new animation will be linearly blended with the currently playing animation, whose weight will decrease during that duration until it reaches `0.0`.
```

**Review by cart — APPROVED** [review]
That video is nicely edited. Well done!

**Comment by mockersf** [comment]
> That video is nicely edited. Well done!

No video editing except encoding, everything is done in Bevy. Each point of view is rendered to a texture, then I organised them with `bevy_ui`. That way I can use the same timers for the animations and the arrows.

**Comment by cart** [comment]
Coooooool :heart:

**Pull request closed without merging**